tslint.json

```diff
@@ -29,7 +29,7 @@
     "max-classes-per-file": false,
     "max-line-length": [
       true,
-      140
+      256
```

Comment:
So you made 3 changes to the default config.  Is there good reason for changing them?
* For the line length, "140" looks good enough I would say.   In the backend, we set it to 140. Looks like a good trade-off.
* `no-switch-case-fall-through` looks a good way to avoid difficult-to-debug bugs with `/* falls through */` as a way to enforce it for cases where you know what you are doing.
* `no-output-on-prefix`: I have no opinion.

Reply:
- The first was changed because it needs for multiple line test as it was required for frontend design.
- The second was to optimize the code but if you want I can enable it, no problem.
- The third one was for output event emitters with on as prefix. I use this to emphasize the events.